_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2122 @@
+{
```

Comment:
<ul><li>Instead of <code>is done</code> say <code>calculated</code></li><li> Link to an appendix instead of saying details at bottom). </li><li>This section should clearly map to the fields in the table, and its not clear which fields match which definitions</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done.</p>